CashBud/Service/LocalPersistenceServiceMock.swift

```diff
@@ -0,0 +1,22 @@
+CashBud/Service/LocalPersistenceServiceMock.swift
+
+// Import necessary modules
+import CashBud.Model.Transaction
+import CashBud.Model.TransactionCreationProcessModel
+
+// Implement LocalPersistenceServiceMock class
```

Comment:
Remove all current functions in the LocalPersistenceServiceMock. Please implement every functions signature according to the protocol and a body so every function can be testes for success and failure.